emcc.py

```diff
@@ -503,7 +503,7 @@ def run(args):
   if '--version' in args:
     try:
       revision = run_process(['git', 'show'], stdout=PIPE, stderr=PIPE, cwd=shared.path_from_root()).stdout.splitlines()[0]
-    except:
+    except CalledProcessError:
```

Comment:
Is this really the only thing that can go wrong here?  What if git is not found isn't that a differnt error type?

Reply:
No, it's not. I already fixed this after the CI failed.